frontend/global/base.css

```diff
@@ -6,7 +6,7 @@
 
 html, body {
     font-size: var(--font-size);
-    line-height: var(--spacing-unit);
+    line-height: 2*var(--spacing-unit);
```

Comment:
What’s the purpose of this?

This isn’t valid CSS. If you actually wanted to multiply by two you’d have to do `calc(2 * var(--spacing-unit))` or, even better, use `var(--spacing-unit-xl)` (which resolves to `calc(2 * var(--spacing-unit))`.

But I don’t think you actually wanted to increase global line height like this?

Reply:
To increase the space between the lines in the `<h1>`. It *does* have an effect.

![Screenshot - 2020-04-30T150548 424](https://user-images.githubusercontent.com/2110676/80713714-2e337900-8af4-11ea-89bd-558bc66c8c41.png)


![Screenshot - 2020-04-30T150635 133](https://user-images.githubusercontent.com/2110676/80713722-312e6980-8af4-11ea-9947-4e429882a38b.png)


Anyways, I'm using `c 'page_header'` now.

Reply:
Ah, that’s true and interesting. I think what’s happening is that the value is invalid and the rule does not have an effect and in consequence, it falls back to the line height defined in the default browser style sheet (`line-height: normal`) which is usually specified relative to the elements font size.